spec/features/admin/multilingual_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:admin_role) { Spree::Role.find_or_create_by!(name: 'admin') }
   let(:admin_user) { create(:user) }
 
-  background do
+  before do
```

Comment:
Great find! I forgot about this syntax, another feature-spec specific naming that is not necessary.